tests/python/test_field.py

```diff
@@ -99,6 +99,37 @@ def test_scalr_field_from_numpy_with_mismatch_shape(dtype, shape):
         x.from_numpy(arr)
 
 
+@pytest.mark.parametrize('dtype', data_types)
+@pytest.mark.parametrize('shape', field_shapes)
+@ti.test(arch=ti.get_host_arch_list())
+def test_field_copy_from(dtype, shape):
+    x = ti.field(dtype, shape)
+    other = ti.field(dtype, shape)
+    x.copy_from(other)
+
+
+@pytest.mark.parametrize('dtype', data_types)
+@pytest.mark.parametrize('shape', field_shapes)
+@ti.test(arch=ti.get_host_arch_list())
+def test_field_copy_from_with_non_field_object(dtype, shape):
```

Comment:
for these two error checking tests you don't have to parametrize the tests ;) Simply test one case is good enough.

Reply:
Thanks for reviewing, It will be removed.